backend/src/routes/api/nb-events/eventUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { V1EventList } from '@kubernetes/client-node';
+import { KubeFastifyInstance, NotebookStatus } from '../../../types';
+
+export const getNotebookEvents = async (
+  fastify: KubeFastifyInstance,
+  nbName: string,
+): Promise<NotebookStatus> => {
+  const response = await fastify.kube.coreV1Api
+    .listNamespacedEvent(
+      fastify.kube.namespace,
+      undefined,
+      undefined,
+      undefined,
+      `involvedObject.kind=Notebook,involvedObject.name=${nbName}`,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      false,
+    )
+    .then((res) => {
+      return res.body as V1EventList;
+    })
+    .catch((e) => {
```

Comment:
We don't need to catch the error here IMO. The error will be caught by the catch here https://github.com/opendatahub-io/odh-dashboard/pull/330/files#diff-97d842340dde487037d9347769f0bab0d4bf247ca2007270aa8a1c609c38573eR13-R15

Reply:
Yeah, I think that's right -- we should avoid catching. If it's a 404 (for instance), nothing else matters in this function.

The client can infer a failed call as the lack of available events no matter what happened.

Reply:
Added a new 404 error in a try catch block and letting the index.ts catch send it.